src/editor/Core/ConfigurationDefaults.re

```diff
@@ -1,3 +1,45 @@
+/*
+ * ConfigurationDefaults.re
+ *
+ * Configuration defaults in string form to generate default configuration from.
+ */
+
+let getDefaultConfigString = configName =>
+  switch (configName) {
+  | "configuration.json" =>
+    Some(
+      {|
+{
+  "editor.minimap.enabled": true,
+  "editor.insertSpaces": false,
+  "editor.indentSize": 4,
+  "editor.tabSize": 4,
+  "editor.DetectIndentation": true,
+  "editor.LargeFileOptimizations": true,
+  "editor.matchBrackets": true,
+  "editor.minimap.enabled": true,
+  "editor.minimap.showSlider": true,
+  "editor.minimap.maxColumn": 120,
+  "editor.lineNumbers": "on",
+  "editor.insertSpaces": false,
+  "editor.indentSize": 4,
+  "editor.tabSize": 4,
+  "editor.renderIndentGuides": true,
+  "editor.highlightActiveIndentGuide": true,
+  "editor.renderWhitespace": "all",
+  "workbench.activityBar.visible": true,
+  "workbench.editor.showTabs": true,
+  "workbench.sideBar.visible": true,
+  "workbench.statusBar.visible": true,
+  "workbench.iconTheme": "vs-seti",
+  "files.exclude": ["_esy", "node_modules"],
+  "editor.zenMode.hideTabs": true
```

Comment:
For now, I've extended this to the full config, mainly to make the test comparison much easier.
It does also make discoverability a little easier whilst the config story is pretty basic.

I can change it to a smaller one if needed though?

Reply:
This sounds good! We can tweak it down the road, and as you mentioned, it helps discoverability for now.